src/schema/show.ts

```diff
@@ -546,6 +546,11 @@ export const ShowType = new GraphQLObjectType({
         return results
       },
     },
+    openingReceptionText: {
+      type: GraphQLString,
+      description:
+        "Alternate Markdown-supporting free text representation of the opening reception event’s date/time",
+    },
```

Comment:
I assume that the previous snake_case field worked because the data coming from the upstream service uses that format, no? If so, I'd expect there to need to be a resolver that pulls out the snake_case field.

Reply:
Yeesh, of course. Thanks!

Reply:
Thanks for catching that silly error. My feeling is that trivial field definitions (i.e. the ones you don’t write resolvers for) don’t need tests, but maybe this one does now, since it transforms data from an upstream response?

Is that a reasonable rule of thumb?

Reply:
I think this is also just a bit too simple to warrant writing a test for, you're just testing JS syntax not business logic

Reply:
Cool, works for me. I think this is done then, if one of y'all will do the honors. Else I'll merge in a bit.

Reply:
I agree 👍